### DISCUSSION TITLE
Error blynklib.py in micrpython 1.23

### DISCUSSION THREAD

**Discussion by ayresnet** [micropython/micropython#15250] [ESP32]
Hello my name es Daniel from Argentina, and I have a problem that I describe below with the latest update of micropython 1.23 on an ESP32.

Connecting to blynk.cloud:443…
Traceback (most recent call last):
File “main.py”, line 167, in
File “blynklib.py”, line 214, in init
File “blynklib.py”, line 81, in init
File “blynklib.py”, line 239, in connect
AttributeError: ‘module’ object has no attribute ‘create_default_context’

If I use micropython 1.22 it works but in the new update of micropython 1.23 blynklib.py no longer works and shows the error above, what do I do? aid. because if I update one day it won’t work.

HELP

**Comment by bikeNomad** [comment]
Perhaps you need to make this change: https://github.com/vshymanskyy/blynk-library-python/pull/82
